dolphinscheduler-meter/src/main/java/org/apache/dolphinscheduler/meter/metrics/BaseServerLoadProtection.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.meter.metrics;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Data
+public class BaseServerLoadProtection implements ServerLoadProtection {
+
+    protected boolean enabled = true;
+
+    protected double maxSystemCpuUsagePercentageThresholds = 0.7;
+
+    protected double maxJvmCpuUsagePercentageThresholds = 0.7;
+
+    protected double maxJvmMemoryUsagePercentageThresholds = 0.7;
+
+    protected double maxSystemMemoryUsagePercentageThresholds = 0.7;
+
+    protected double maxDiskUsagePercentageThresholds = 0.7;
+
+    @Override
+    public boolean isOverload(SystemMetrics systemMetrics) {
+        if (!enabled) {
+            return false;
+        }
+        if (systemMetrics.getSystemCpuUsagePercentage() > maxSystemCpuUsagePercentageThresholds) {
+            log.info(
+                    "OverLoad: the system cpu usage: {} is over then the maxSystemCpuUsagePercentageThresholds {}",
+                    systemMetrics.getSystemCpuUsagePercentage(), maxSystemCpuUsagePercentageThresholds);
+            return true;
+        }
+        if (systemMetrics.getJvmCpuUsagePercentage() > maxJvmCpuUsagePercentageThresholds) {
+            log.info(
+                    "OverLoad: the jvm cpu usage: {} is over then the maxJvmCpuUsagePercentageThresholds {}",
+                    systemMetrics.getJvmCpuUsagePercentage(), maxJvmCpuUsagePercentageThresholds);
+            return true;
+        }
+        if (systemMetrics.getJvmMemoryUsedPercentage() > maxJvmMemoryUsagePercentageThresholds) {
```

Comment:
I'm not sure if judging a jvm's memory usage is necessary. I think the heap memory of JVM will always increase over time. After reaching certain conditions, minor/major GC will be triggered and the memory usage will decrease. 

Therefore, I think that the temporary increase in jvm memory utilization is not abnormal. But due to memory leaks and other reasons, it is indeed an anomaly that the JVM's memory utilization remains high for a long time.

Reply:
Good catch, I have removed this setting, PTAL